e2e/test/scenarios/metrics/reproductions/metrics-reproductions.cy.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import { createQuestion, restore } from "e2e/support/helpers";
+
+const { ORDERS_ID, ORDERS } = SAMPLE_DATABASE;
+
+describe("issue 47058", () => {
+  it("should show the loading page while the question metadata is being fetched (metabase#47058)", () => {
+    restore();
+    cy.signInAsNormalUser();
+    cy.intercept("GET", "/api/card/*/query_metadata", req =>
+      req.continue(() => new Promise(resolve => setTimeout(resolve, 1000))),
+    ).as("metadata");
+
+    createQuestion({
+      name: "Metric 47058",
+      type: "metric",
+      query: {
+        "source-table": ORDERS_ID,
+        aggregation: [["count"]],
+      },
+    }).then(({ body: { id: metricId, name: metricName } }) => {
+      createQuestion({
+        name: "Question 47058",
+        type: "question",
+        query: {
+          "source-table": ORDERS_ID,
+          fields: [
+            ["field", ORDERS.ID, {}],
+            ["field", ORDERS.TOTAL, {}],
+          ],
+          aggregation: [["metric", metricId]],
+          limit: 1,
+        },
+      }).then(({ body: { id: questionId } }) => {
+        cy.visit(`/question/${questionId}/notebook`);
+
+        cy.findByText("Loading...").should("be.visible");
```

Comment:
Please unnest all code starting from this line until the end of the function, otherwise ESLint warnings are not kicking in.